spec/models/message_spec.rb

```diff
@@ -17,41 +17,48 @@
 require 'spec_helper'
 
 describe Message do
-	let!(:user) { create :user }
-
-	describe 'create' do
-		let(:message) { build :message }
-		it 'creates a new message' do
-			expect(message.valid?).to be_true
-			expect do
-				message.save!
-			end.to change(Message, :count).by(1)
-		end
-	end
-
-	describe 'Permissions' do
-		let(:message) { Message.new }
-		describe 'can_create?' do
-			it 'returns true for user' do
-				expect(message.can_create?(user)).to be_true
-			end
-			it 'returns false if user is banned' do
-				create :ban, :mute, user: user
-				expect(message.can_create?(user)).to be_false
-			end
-		end
-
-		describe 'can_show?' do
-			let!(:message) { create :message }
-			it 'returns true if sender' do
-				expect(message.can_show?(message.sender)).to be_true
-			end
-			it 'returns true if receiver' do
-				expect(message.can_show?(message.recipient)).to be_true
-			end
-			it 'returns false if neither sender nor receiver' do
-				expect(message.can_show?(user)).to be_false
-			end
-		end
-	end
-end
\ No newline at end of file
+  let!(:user) { create :user }
+
+  describe 'create' do
+    let(:message) { build :message }
+
+    it 'creates a new message' do
+      expect(message.valid?).to eq(true)
+      expect do
+        message.save!
+      end.to change(Message, :count).by(1)
+    end
+  end
+
+  describe 'Permissions' do
+    let(:message) { Message.new }
+
+    describe 'can_create?' do
+      it 'returns true for user' do
+        expect(message.can_create?(user)).to be_truthy
+      end
+
+      it 'returns false if user is banned' do
+        create :ban, :mute, user: user
+        expect(message.can_create?(user)).to be_falsey
+      end
+    end
+
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.